packages/core/src/index.ts

```diff
@@ -169,30 +170,32 @@ export const createConversation = (config: Config): ConversationHandler => {
   };
 
   const messageResposeHandler = (response: any) => {
-    setState({
-      conversationId: response.conversationId,
-      responses: [
-        ...state.responses,
-        {
-          type: "bot",
-          receivedAt: new Date().getTime(),
-          payload: {
-            conversationId: response.conversationId,
-            messages: response.messages.map((message: any) => ({
-              messageId: message.messageId,
-              text: message.text,
-              choices: message.choices || [],
-            })),
-            metadata: response.metadata,
-            payload: response.payload,
-            context: response.context,
+    if (response && !response.isPending) {
+      setState({
+        conversationId: response.conversationId,
+        responses: [
+          ...state.responses,
+          {
+            type: "bot",
+            receivedAt: new Date().getTime(),
+            payload: {
+              conversationId: response.conversationId,
+              messages: response.messages.map((message: any) => ({
+                messageId: message.messageId,
+                text: message.text,
+                choices: message.choices || [],
+              })),
+              metadata: response.metadata,
+              payload: response.payload,
+              context: response.context,
+            },
           },
-        },
-      ],
-    });
+        ],
+      });
+    }
   };
 
-  const sendToBot = (body: any) => {
+  const sendToBot = (body: any): Promise<globalThis.Response> => {
```

Comment:
🤷  was responding to a ts-lint complaint.  We're not strict checking on this repo, could take this back out if you wish.

Reply:
I don't use tslint :/ not sure what this `globalThis` is but as long as the regular build is happy there is no harm in keeping it in.